fixtures/api/list_attribute_extend/data.json

```diff
@@ -179,7 +179,7 @@
                 "en": "\/listAttributesExtend\/attribute id"
             },
             "description": {
-                "en": "It is used for modification only."
+                "en": "It is used for modification only. Properties marked as arrays ([]) submitted with the PUT method are overwritten, so only the submitted items will remain. A potential issue is that non-submitted list items may disappear."
```

Comment:
> A potential issue is that non-submitted list items may disappear.

Ez így zavaróan tág értelmű. Ha a be nem küldött elemek törlődni fognak, akkor jelentsük ki azt. Érted, ne úgy legyen megfogalmazva, hogy "a be nem küldött elemek eltűnhetnek" - most akkor eltűnnek vagy nem? :D Ráadásul az adatbázisból nem szoktak eltűnni dolgok, az valami inkonzisztens hibát sejtetne, hanem törölni szokták őket szándékosan. Plusz ez nem lehetséges probléma, mint fentebb a már létező elem - ami tényleg nem garantált, csak lehetséges - hanem szimplán tény, hogy ez fog történni. Gondolom a chatgpt javasolta így, de nem találta el.

Reply:
Köszi, jogos, javítottam: https://github.com/Shoprenter/sr-api-docs/pull/135/commits/02d5565ff2f1d36b90a54f315e1c4e8dcf3f4f99